ext/Pod-Html/t/pod2html-lib.pl

```diff
@@ -41,20 +45,40 @@ sub convert_n_test {
     my $infile   = catpath $vol, $new_dir, "$podfile.pod";
     my $outfile  = catpath $vol, $new_dir, "$podfile.html";
 
-    my %default_args_table = (
+    my %args_table = (
         infile      =>    $infile,
         outfile     =>    $outfile,
         podpath     =>    't',
         htmlroot    =>    '/',
         podroot     =>    $cwd,
     );
-    my @default_args_list = map { "--${_}=" . $default_args_table{$_} }
-        keys %default_args_table;
-    # To add/modify args to p2h, use @p2h_args
-    Pod::Html::pod2html(
-        @default_args_list,
-        @p2h_args,
-    );
+    my %no_arg_switches = map { substr($_,2) => 1 } ( qw|
+           --flush --recurse --norecurse
+           --quiet --noquiet --verbose --noverbose
+           --index --noindex --backlink --nobacklink
+           --header --noheader --poderrors --nopoderrors
+    | );
```

Comment:
Apropos of nothing, I am surprised by this `map { substr($_,2) => 1 }` form, when you could, I think, just drop the `--` from the source entirely.  It's not a problem, but seems like a small amount more code than is needed.

Reply:
> Apropos of nothing, I am surprised by this `map { substr($_,2) => 1 }` form, when you could, I think, just drop the `--` from the source entirely. It's not a problem, but seems like a small amount more code than is needed.

@rjbs, thanks for taking a look at this.  I'm a bit unclear as to what you're referring to as "the source" here.  Are you referring just to the contents of `%no_arg_switches` here?  Or are you referring to the strings passed as arguments to `Pod::Html::html()`?

Thank you very much.
Jim Keenan

Reply:
What I meant was "remove the double dashes from the strings in the qw(...) operator, and then you won't need to remove them in the map."  To me, this reads like "well, earlier on I was using `$_` itself also in the map, but then I removed it, leaving the code with more complexity than required.

As I said, this isn't really a problem, but feels like a bit of a vestigial appendix that could be excised.